src/app/choice/components/choice/choice.component.ts

```diff
@@ -7,14 +7,16 @@ import { FirebasePlayerService } from '../../../firebase/services/firebase-playe
   templateUrl: './choice.component.html',
   styleUrls: ['./choice.component.scss']
 })
-export class ChoiceComponent {
+export class ChoiceComponent implements OnInit {
   choiceEnum = Choice;
   choice: Choice | undefined;
-
   constructor(private playerService: FirebasePlayerService) {}
 
+  ngOnInit(): void {
+    this.playerService.valueChanges().subscribe(player => this.choice = player?.choice);
```

Comment:
maybe under the constructor instead of `ngOnInit()`?

Reply:
Any particular reason why? I have a feeling that `onInit` might be preferable but I am not sure why :)

Reply:
Any Particular reason why `ngOnInit() `?

it is basically about initiating a  class member no big logic behind it, why have extra lines with ngOnInit() ?
We are doing the same thing in `room.component.ts`  and `oppenent.compoenent.ts` so maybe for consistency sake :)